.github/workflows/aws.yml

```diff
@@ -0,0 +1,28 @@
+name: aws
+on: push
+
+# on:
+#   push:
+#     branches:
+#       - 'dev'
```

Comment:
You might want to change that, depending on when to trigger the job. Most l likely on PRs to `dev` I guess?

Reply:
Yes, you are correct.  I will be discussing the details of this with @rickoco (to understand our legal constraints) and the Members (to understand their expectations) and will update the trigger accordingly.

Reply:
I think it relates to the process which had been setup for cv32e40p: https://github.com/openhwgroup/cv32e40p#contributing